README.md

```diff
@@ -7,122 +7,86 @@
 
 CLI for the Griptape Framework and Cloud.
 
-## Prerequisites
```

Comment:
Why did we remove this?

Reply:
We no longer have a `cloud` command. The only command available is the local emulator via `gt server` which does not require a Cloud account.

Reply:
It looks like there's a requirement for a github account though, right?